src/patternfly/demos/Toolbar/examples/toolbar-attribute-value-single-select-filter-example.hbs

```diff
@@ -0,0 +1,75 @@
+{{#> data-toolbar data-toolbar--id="data-toolbar-attribute-value-single-select-filter-example"}}
+  {{#> data-toolbar-content}}
+    {{#> data-toolbar-group data-toolbar-group--modifier="pf-m-toggle-group pf-m-reveal-on-lg" data-toolbar-group--IsExpanded="false"}}
+      {{> data-toolbar-toggle}}
+      {{#> data-toolbar-group data-toolbar-group--modifier="pf-m-filter-group"}}
+      {{#> data-toolbar-item}}
+        {{#> select id="select-two" select-toggle--icon="fas fa-filter"}}
```

Comment:
Same with creating the `id` for this using `data-toolbar--id` and make it the same between demos